### ISSUE TITLE
Progress bar bug, e.g. downloaded 5.8/4.9 GB

### ISSUE THREAD

**Issue by cuducos** [cuducos/minha-receita#53]
Sometimes the progress bar for the `download` command in the `dev` branch goes beyond the expected total:

```
Downloading 100% |██████████████████████████████████████| (5.8/4.9 GB, 370.135 kB/s)
```

Note the 5.8 GB out of 4.9 GB.

Probably this is because some downloads fail, then we restart them from the beginning but keep the downloaded bytes in the progress bar counter.

I won't fix that or review PRs related to that until #37 is over, though — either way, feel free to contribute and I get back to code or code review once the priority is covered : )

**Comment by vidalvasconcelos** [comment]
Perhaps we could use a progress bar per file, this would make it more explicit which files are being downloaded and which ones will be problematic collaterally solving this issue. 

What do you think about?

**Comment by cuducos** [comment]
Since we download a lot of files (~40 files), I think that having this amount of progress bars on the screen would be an ugly and confusing UI. I tend to go with this philosophy for the CLI:

* On the one hand, provide the simplest way to accomplish a task (e.g. `minha-receita download` starts the download, surfaces the overall progress and gets the job done)
* On the other, offer tools to workaround when the simplest way fails, isolating what the command does in a UNIX philosophy way, i.e. do one thing and do one thing well (e.g. `minha-receita download --urls-only` list the URLs to be downloaded, and the user can pipe them to `curl -O`, `wget` or a custom script)

**Comment by cuducos** [comment]
Closed by #86
